charts/yetibot/Chart.yaml

```
@@ -24,7 +24,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time
 # you make changes to the chart and its templates, including the app version.
-version: 1.0.58
+version: 1.0.59
```

This gets auto bumped by automation ([example commit](https://github.com/yetibot/yetibot-helm/commit/e680128567ad08bec98b5970a10a5f0b765f36b7))  so we can leave it at `.58` in the PR.

```suggestion
version: 1.0.58
```

---

Wait I'm wrong - that's only when bumping deps.